404.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```

Comment:
Try with a clone of https://github.com/fybrik/fybrik/blob/site/dev/404.html + meta tags at the top.

Reply:
It seems to work so I used that.